pangeo_forge_recipes/recipes/xarray_zarr.py

```diff
@@ -16,6 +16,8 @@
 import dask
 import fsspec
 import numpy as np
+import scipy
+import scipy.stats
```

Comment:
Scipy is a pretty heavy dependency. I'm not sure it's worth bringing on board just to generate a log message.